src/main/webapp/resources/js/components/typography/CodeBlock.jsx

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export function CodeBlock({ children }) {
```

Comment:
Add a comment here on when to use etc...

Reply:
Added in 39c89d161e41dc4241d4d58b6333be75bf9d2a92